packages/atomic/src/components/commerce/atomic-commerce-text/atomic-commerce-text.tsx

```diff
@@ -32,6 +32,7 @@ export class AtomicCommerceText
   @Prop({reflect: true}) public value!: string;
   /**
    * The count value used for plurals.
+   * @type {number}
```

Comment:
This is necessary for the url manager of storybook to work 🤔. We will have to add jsdoc to every prop.

Reply:
Good JSDOC ain't a bad thin :) 
It helps with Intellisense
Custom Element Manifest may also be packed up to help.

tldr: good jsdoc is good.